bitnami/spark/templates/statefulset-worker.yaml

```
@@ -147,6 +147,11 @@ spec:
             - name: http
               containerPort: {{ .Values.worker.webPort }}
               protocol: TCP
+            {{- if .Values.security.ssl.enabled }}
+            - name: https
+              containerPort: {{ .Values.worker.webPortHttps }}
+              protocol: TCP
+            {{- end }}
```

Please use the same logic as the master Statefulset.

```suggestion
            {{- if not .Values.security.ssl.enabled }}
            - name: http
              containerPort: {{ .Values.worker.webPort }}
              protocol: TCP
            {{- else }}
            - name: https
              containerPort: {{ .Values.worker.webPortHttps }}
              protocol: TCP
            {{- end }}
```